client/src/tutorials/Template_counter/pages/2.md

```diff
@@ -0,0 +1,19 @@
+# Structure of the Program 
+
+As you probably have seen, our initial program is divided in 3 main sections: Functions, Data Validators and Data Structures.
+
+- **Smart Contract Functions**: a section with the operations and logic of the program. 
+
+- **Data Validators**: a section for all the accounts that we will have to create and declare. As you can see, the *Context*<Name> from the functions is actually the same as the Data validators.
+
+- **Data Structures**: the structure of the accounts / rust structs that we will use to store info! In the program is pretty clear, the Counter account will store 2 things:
+```bash
```

Comment:
`bash` should be `rust`

Reply:
Also ideally, the images live under the tutorial's folder, not the existing tutorial template folder. You can create `counter-easy` folder inside `public/tutorials` to keep things more organized.